help/en/releasenotes/current-draft-note.shtml

```diff
@@ -141,7 +141,17 @@
 
     <h3>New / Updated decoder definitions</h3>
         <ul>
-          <li></li>
+          <li>The decoder definitions have been updated to make 
+            CV19 "Consist Address" and "Consist Direction"
+            read-only quantities.  This lets you read and see
+            the value in the decoder, but prevents a Write ALl
```

Comment:
Does this line need to be updated again?

Reply:
Done.